picky/src/pkcs12/encryption.rs

```diff
@@ -28,12 +29,12 @@ impl Pkcs12CryptoContext {
     pub fn new_with_password(password: &str) -> Self {
         Self {
             password: password.to_string().into(),
-            rng: Box::new(rand::rngs::OsRng),
+            rng: Box::new(ChaCha20Rng::from_os_rng()),
```

Comment:
`Pkcs12CryptoContext` uses a random as `Box<dyn Rng>`(type-erased), and `OsRng` is no longer dyn-compatible, so we need to use `ChaCha20Rng`.